core/amber/src/main/python/core/models/operator.py

```diff
@@ -47,6 +48,25 @@ def close(self) -> None:
         """
         pass
 
+    def process_state(self, state_: State, port: int) -> State:
```

Comment:
remove `_` from `state_`

Reply:
This one is aligned with process_tuple API.

Reply:
The reason for `tuple_` is to differentiate it from `tuple`, which is a python built-in keyword. 

There is no conflict for `state` so we don't need to add the `_`.

Reply:
Renamed.